src/index.html

```diff
@@ -11,6 +11,26 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+
+    <form class="search-form search-form_big" data-qa="big">
+      <label class="search-form__label search-form__label_big">
+        <input data-qa="keypress"
+        class="search-form__input search-form__input_big"
+        type="text"
+        placeholder="Try &quot;Los Angeles&quot;"
+        >
+      </label>
+    </form>
+
+    <form class="search-form search-form_small" data-qa="small">
```

Comment:
```suggestion
    <form class="search-form search-form--small" data-qa="small">
```
it is more common naming for modifiers classes